index.html

```diff
@@ -100,14 +109,15 @@ <h3>Join our Slack community!</h3>
           var warningOpen =
             'You must agree to our code of conduct by checking the boxes above. <a href="#wontAbuse" role="button" draggable="false">Go to first checkbox</a>';
 
-          var wontAbuse = document.getElementById("wontAbuse");
-          var wontSpam = document.getElementById("wontSpam");
-          var willUphold = document.getElementById("willUphold");
+          var formInputs = document.querySelectorAll("#signup input");
           var signupBtn = document.getElementById("signupBtn");
           var formWarning = document.getElementById("formWarning");
 
           var validate = function () {
-            if (wontAbuse.checked && wontSpam.checked && willUphold.checked) {
+            const index = Array.from(formInputs).findIndex(
+              (el) => el.checked === false
+            );
+            if (index === -1) {
```

Comment:
This could be simplified as a querySelector that looks for unchecked checkboxes within the form and counts them.

```suggestion
            const unchecked = document.querySelectorAll('#signup input[type=checkbox]:not([checked])')
            if (unchecked.length === 0) {
```